tests/functional/test_single_model_vehicle.py

```diff
@@ -14,7 +14,9 @@
 # limitations under the License.
 #
 import sys
-
+import cv2
```

Comment:
If we need this package, OpenCV should be included in requirements-dev.txt.
Although you can consider using tensorflow functions for image processing as it's already our dependency and we wouldn't need to include OpenCV.
Default preprocessing function uses tensorflow, you could check it out and see if you could apply it here.

https://github.com/openvinotoolkit/model_server/blob/ams/extras/ams_wrapper/src/preprocessing/preprocess_image.py

Reply:
We will use tf and preprocess when we move tests to ams_wrapper if that ok.